lib/services/address/encoding.js

```diff
@@ -19,6 +19,15 @@ exports.encodeSpentIndexSyncKey = function(txidBuffer, outputIndex) {
   return key.toString('binary');
 };
 
+exports.encodeMempoolAddressIndexKey = function(hashBuffer, hashTypeBuffer) {
+  var key = Buffer.concat([
+    hashBuffer,
+    hashTypeBuffer,
+  ]);
+  return key;
```

Comment:
`key.toString('binary')` would be more direct here, as it looks like `key.toString()` is being called when used as a key in the object.